gsoc2018-bharat/src/tsne.py

```diff
@@ -0,0 +1,33 @@
+import numpy as np
+from sklearn.manifold import TSNE
```

Comment:
make this a cli tool that takes the file you want to display instead of hardcoding the path